app/routes/__frontend/resources/developer-resources/developer-health/burnout.mdx

```diff
@@ -0,0 +1,101 @@
+---
+meta:
+  title: Burnout
+  description: A collection of guides to help with developer burnout.
+hero:
+  Hero: UndrawMeditating
+tags:
+  - memberresources
+  - memberresourcesIndex
+order: 1
+---
+
+import TextContainer from '~/components/content/TextContainer';
+import LeadText from '~/components/content/LeadText';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+Developer burnout is an experience that’s too prevalent in the tech industry, often resulting from workplace stress, a lack of psychological safety in the workplace, negative experiences or feedback, and results in exhaustion, an inability to complete work tasks effectively, a lack of desire to work or ambivalence towards work or teammates.
```

Comment:
Developer burnout is an experience that’s too prevalent in the tech industry. It often results from workplace stress, a lack of psychological safety in the workplace, or negative experiences or feedback. The results include exhaustion, an inability to complete work tasks effectively, a lack of desire to work or ambivalence towards work or teammates, and more.